API/wardrobe.py

```diff
@@ -12,9 +12,42 @@
 router = APIRouter(prefix="/wardrobe")
 
 
+
+import datetime
+import os
+import sys
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from API.database import DB, get_database
+from Common.user_functions import (
+    create_access_token,
+    expires_in,
+    get_current_user,
+    verify_password,
+    get_password_hash,
+)
+from Models.user import User as Model_User
+from Validators.user import User, UserLogin, EditUser
+
+from Models.collection import Collection
+
+
 @router.post("/connect")
-def user_connect(wadrobe_code: int, user: User = Depends(get_current_user)):
-    return {}
+def user_connect(user: UserLogin):
```

Comment:
dane usera można wyciągnąć z tokena: user: User = Depends(get_current_user)
i tak user powinien być zalogowany żeby móc dodać szafę, więc jedynie powinien przekazywać nr szafy:
W skrócie lepiej będzie chyba to przerobić na poprzednie wyjście, jedynie int zmienić na str